website/blog/2023-11-03-release-1.5.md

```diff
@@ -0,0 +1,219 @@
+---
+title: Release Notes - Podman Desktop 1.5
+description: Podman Desktop 1.5 has been released!
+slug: podman-desktop-release-1.5
+authors: duffy
+tags: [podman-desktop, release, kubernetes, openshift, onboarding]
+hide_table_of_contents: false
+image: /blog/img/podman-desktop-release-1.5/onboarding-selkies.png
+---
+
+import ReactPlayer from 'react-player'
+
+Podman Desktop 1.5 Release! 🎉
+
+With this release of Podman Desktop, we're introducing **a new onboarding feature** that we hope will earn your seal of approval! But wait... there's so much more!
+
+- **Onboarding**: Guided setup and configuration of **Podman** and **Compose**
+- **Podman 4.7.2**: [Podman 4.7.2](https://github.com/containers/podman/releases) is now included in Windows and Mac installers
+- **Command Palette**: Gain easy access to various commands via a new keyboard-driven command palette
+- **Expanded "Summary" tab for Kubernetes pods**: Go deep with extended details on Kuberentes pods in the pod "Summary" tab
+- **Environment file support**: Chart out environment variables for new containers to access on creation
+- **Enhancements to the Settings area**: Get your bearings with improved Docker compatibility mode controls
+- **Improved user experience for state changes**: No more dead reckoning on container state with improved visual indication of status
+- **Extension API improvements**: A boatload of improvements to the extension API enabling more goodness from Podman Desktop's extensions
+
+Podman Desktop 1.5 is now available. [Click here to download it](/downloads)!
+
+![Podman-desktop-1-5-hero](img/podman-desktop-release-1.5/onboarding-selkies.png) -->
```

Comment:
Extraneous `-->`, shows up on final page.

Reply:
fixed, thank you!